automated/linux/hci-smoke/hci-smoke-test.sh

```diff
@@ -123,7 +135,17 @@ test_hciconfig
 test_hciconfig_boot
 test_hciconfig_down
 test_hciconfig_up
-test_hcitool_scan
+if [ "${MODE}" = "both" ] || [ "${MODE}" = "normal" ]; then
+    test_hcitool_scan
+else
+    report_skip "hciconfig-scan"
```

Comment:
is skip really the right status? looks like the user decided not to even execute the test, so it's not 'skipped' but simply not executed. so it should not show up in the test results. I think skipped would imply that we wanted to run it, but because of 'something' we couldn't run it. no?

Reply:
I prefer to have all possible tests listed always (for easier accounting) but I'm OK removing this part if you think it's a bad idea.

Reply:
well.. you know that better than me ;) i found it odd, but if this is how we typically do it, then it's fine for me!